lib/ClimaLandSimulations/experiments/ozark.jl

```diff
@@ -1,6 +1,21 @@
 using ClimaLandSimulations
+using ClimaLandSimulations.Fluxnet
 
-sv, sol, Y, p, inputs, climaland = fluxnet_simulation("US-MOz");
+# default parameters, except for custom parameter hetero_resp.b
+sv_test, sol_test, Y_test, p_test = fluxnet_simulation(
+    "US-MOz";
+    params = ozark_default_params(; hetero_resp = hetero_resp_ozark(; b = 2)),
+)
+
+# default parameters
+sv, sol, Y, p = fluxnet_simulation("US-MOz")
+
+# add custom param
```

Comment:
can remove commented lines?

Reply:
why? the comments is to explain a bit this "experiment" code

This file should eventually be a unit test file though, it is just testing functionality... 

Let's think a bit more what to do in next PRs (what should be unit test, what could be an experiment)